samples/tracing/topology.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rmqbroker
+---
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: cluster
+  namespace: rmqbroker
+spec:
+  replicas: 1
+---
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: default
+  namespace: rmqbroker
+  annotations:
+    eventing.knative.dev/broker.class: RabbitMQBroker
+spec:
+  config:
+    apiVersion: rabbitmq.com/v1beta1
+    kind: RabbitmqCluster
+    name: cluster
+    namespace: rmqbroker
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: subscriber
+  namespace: rmqbroker
+spec:
+  template:
+    spec:
+      containers:
+      - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: default
+  namespace: rmqbroker
+spec:
+  broker: default
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: subscriber
+      namespace: rmqbroker
+---
+apiVersion: sources.knative.dev/v1
+kind: ContainerSource
+metadata:
+  name: heartbeats
+  namespace: rmqbroker
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-nightly/knative.dev/eventing/cmd/heartbeats:latest
+          name: heartbeats
+          args:
+            - --period=1
+          env:
+            - name: POD_NAME
+              value: "heartbeats"
+            - name: POD_NAMESPACE
+              value: "rmqbroker"
+            - name: K_CONFIG_TRACING
+              value: '{"backend":"zipkin","debug":"true","zipkin-endpoint":"http://otel-collector.observability:9411/api/v2/spans"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: default
+      namespace: rmqbroker
```

Comment:
Could we split this large topology definition into multiple files? It's easier to understand the building blocks when looking at files, and it's also easier to run diffs against various samples (also easier to repurpose & reuse).